pkg/memory/repository.go

```diff
@@ -0,0 +1,69 @@
+package memory
+
+import (
+	"fmt"
+	"github.com/goharbor/harbor-scanner-clair/pkg/job"
+	"github.com/goharbor/harbor-scanner-clair/pkg/model/harbor"
+	"sync"
+)
+
+type repository struct {
+	data map[string]job.ScanJob
+	sync.Mutex
+}
+
+// TODO Replace with thread safe cache with TTL for scan reports
```

Comment:
Replace with `sync.Map`?

Reply:
We have Redis now for persisting scan jobs / scan reports. Reports are saved with TTL.